_schema.graphql

```diff
@@ -3035,6 +3035,64 @@ type AuctionResultPriceRealized {
   ): String
 }
 
+# An auction lot result
+type AuctionResultsByArtists {
```

Comment:
I see that the corresponding diffusion PR has been merged, but we'll want to be sure to follow the convention around adding `Connection` to the variable name of connections.  (So `AuctionResultsByArtistsConnection`.)

Reply:
I will update Diffusion and this PR tonight, since I'm switching to the bidding engine project next week. Thanks @damassi!